decidim-elections/app/views/decidim/elections/admin/trustees_participatory_spaces/index.html.erb

```diff
@@ -23,7 +23,11 @@
           <% trustees.each do |trustee| %>
             <tr>
               <td>
-                <%= trustee.user.name %>
+                <% if trustee.name %>
```

Comment:
Just a small personal preference. I prefer being more explicit in these checks to avoid unexpected errors. I'd rather use `if trustee.name.present?` instead 😄

Reply:
Sure! 😊 I'll update it.